### ISSUE TITLE
Add author-like field for "adapted by"

### ISSUE THREAD

**Issue by VTOpenLib** [pressbooks/pressbooks#2184]
### Feature Description

Please add a field in addition to the "author" field for inclusion of the name(s) of adapters.

### Feature Use Case

When customizing / adapting an existing openly licensed book, it is important to be able to clearly and prominently display who the original author is (if allowed) and to also include the name of the "adapter" and to indicate their role as having adapted the book. 

As it is, I have one book where the adapter is listed as the author (because the author declined to be listed). This is not a good practice. 

I would like to be able to list this person not as the author but as the adapter. Here are the possible permutations I see: 

Author(s)

Author(s). Adapted by _____

[Blank. author declined to be listed] Adapted by ______

### Other Notes

Open textbooks are designed to be adapted. Let's make it super easy for adapters to honor authors but still get their name in a prominent place!

**Comment by unizami-kpu** [comment]
Yes, please! It would be incredibly useful to have the ability to specifically note adaptations within the chapters of a PB. Perhaps something similar to the media library where authors are able to attribute work and note "adapted by" information .

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 5y 10m ago with no updates in the last 8m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
